widget/ui/src/components/RefreshProgressButton/RefreshProgressButton.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import type { SvgIconPropsWithChildren } from '../SvgIcon';
+import type { Meta } from '@storybook/react';
+
+import React from 'react';
+
+import { RefreshProgressButton } from './RefreshProgressButton';
+
+export default {
+  title: 'Components/RefreshProgressButton',
+  component: RefreshProgressButton,
+  args: {
+    size: 22,
+    color: 'black',
```

Comment:
I'm not sure this is related to you changes or not, but changing `color` value to other colors, doesn't work.

Reply:
I checked it over and there was nothing wrong.
Keep in mind that the props in the documents section cannot be changed.